File: Java/day3.java
public class Solution {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        if(n % 2 == 1) {
            System.out.println("Weird");
        } else if((n >= 2 && n <= 5) || n > 20) {
            System.out.println("Not Weird");
        } else {
            System.out.println("Weird");
        }
    }

}

File: C++/day3.cpp
#include <iostream>

using namespace std;

int main(){
    int N;
    cin >> N;
    if(N % 2 == 1) {
        cout << "Weird";
    } else if ((N >= 2 && N <= 5) || N > 20) {
        cout << "Not Weird";
    } else {
        cout << "Weird";
    }
    return 0;
}
